tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml

```diff
@@ -40,15 +40,20 @@
                             Orientation="Horizontal">
                             <DropDownButton 
                                 Content="{Binding SourceControlProviderDisplayName}"
-                                ToolTipService.ToolTip="{Binding SourceControlProviderPackageDisplayName}">
+                                AutomationProperties.Name="{Binding RepoPathMapping}"
```

Comment:
Does this set the name correctly (does it sound right in Narrator / look right in acc insights/inspect) or don't you need to do the 
`<data name="ButtonName.[using:Microsoft.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">` thing in the resw file?

Reply:
Yes, it does seem to set the name correctly. In narrator, the path is correctly announced with backslashes, followed by the status of the drop down button. I also checked with Accessibility Insights and it did not flag any issues.  Thanks!

Reply:
Setting a binding for the automation name is needed when something has to change between all the items.  For example, in RepoConfig.  The narrator says for the edit button "Edit repository [path]" where path is different for each item.